site/content/en/docs/Installation/_index.md

```diff
@@ -46,48 +46,82 @@ gcloud container clusters create --machine-type n1-standard-2 open-match-dev-clu
 ```
 
 ### Setup a Minikube Cluster
+Minikube is a tool that allows you to run Kubernetes locally. It deploy a single-node cluster inside a VM for local development purpose. Pleae see Kubernetes doc below for tutorials in detailed.
 
 [Set up a Local Minikube cluster](https://kubernetes.io/docs/setup/minikube/)
 
 ## Install Core Open Match
 
-Open Match comprises of a set of services that enable core functionality such as Ticket management, filtering, Match generation etc. It also includes a
-state storage that Open Match uses to persist state needed for its functioning.
+Open Match comprises of a set of services that enable core functionality such as Ticket Management and Filtering etc. It also includes a
+state storage to persist state needed for its functioning.
 
 The simplest way to install Open Match is to use the install.yaml files for the latest release.
 This installs Open Match with the default configuration.
 
 ```bash
 # Create a cluster role binding (if using gcloud on Linux or OSX)
-kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-admin --user `gcloud config get-value account`
+> kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-admin --user `gcloud config get-value account`
```

Comment:
Why these brackets?

It removes the ability to copy paste a block of commands and run them.